tags/java.md

```diff
@@ -1 +1,8 @@
-https://prismlauncher.org/wiki/getting-started/installing-java/
+---
+embed:
+  title: Java Instructions 
+  color: orange
+---
+
+For security reasons, Prism Launcher does not bundle in Java with itself. The instructions to setup and install Java can be found [here](https://prismlauncher.org/wiki/getting-started/installing-java/).
+***DO NOT USE ORACLE!!!***
```

Comment:
why?

Reply:
yeah, why the oracle part. I mean, I prefer FOSS JREs myself, but I don't think there is something inherently wrong with Oracle JDK specifically?

Reply:
I was going to type out a long story about my ever loving hatred for Oracle but then I realized now is not the time or place. Removing that part.